app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command, source_file, directory = command.split()
+    else:
+        return
```

Comment:
You can simplify your code
```suggestion
    if len(command.split()) != 3:
       return
    command, source_file, directory = command.split()
```